xbmc/epg/EpgInfoTag.h

```diff
@@ -85,6 +85,22 @@ namespace EPG
      */
     bool IsActive(void) const;
 
+    /* @brief Check if this event can be recorded
+     * @return True if it can be recorded.
+     */
+    bool IsRecordable(void) const;
+
+    /* @brief Check if this event can be played
+     * @return True if it can be played.
+     */
+    bool IsPlayable(void) const;
+
+    /* @brief Get the stream url for this event
+     * @param properties The properties to be provided to the file item
+     * @return The stream url, empty string if playing is not possible
+     */
+    const std::string GetStreamUrl(CStringPropertyMapPtr &properties) const;
```

Comment:
okay, obviously the properties are filled by this method. Please make the documentation more clear.